src/config/index.ts

```diff
@@ -24,6 +24,7 @@ export const BASE_EXCHANGE_URL = 'https://exchange.pancakeswap.finance'
 export const BASE_ADD_LIQUIDITY_URL = `${BASE_EXCHANGE_URL}/#/add`
 export const BASE_LIQUIDITY_POOL_URL = `${BASE_EXCHANGE_URL}/#/pool`
 export const BASE_BSC_SCAN_URL = 'https://bscscan.com'
+export const BASE_SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/pancakeswap'
```

Comment:
I think this should stay in `endpoints.ts`

Reply:
Sure thing man, have moved it 👍